.github/workflows/msktutil-solaris.yml

```diff
@@ -0,0 +1,40 @@
+name: msktutil Solaris
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build on Solaris
+    strategy:
+      fail-fast: false
+      matrix:
+        cfg:
+          - { cc-version: gcc }
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Build msktutil with ${{ matrix.cfg.cc-version }}
+      id: build
+      uses: vmactions/solaris-vm@v1
+      with:
+        mem: 2048
+        usesh: true
+        prepare: |
+          # basic dependencies
+          pkgutil -y -i gcc5g++ autoconf sasl_gssapi gmake openldap_client
+        run: |
+          export PATH=/opt/csw/bin:$PATH
+          case "${{ matrix.cfg.cc-version }}" in
+            gcc)
+              export CC=gcc
+              export CXX=g++
```

Comment:
Any reason not to use the bundled compiler? GCC 5 is dead, no?

Reply:
Still trying to flesh this out, so tried to start with gcc first. This was the latest version available in CSW, though. I've added another target to use the system's default compiler instead. This one might needs a few tries, still, and I'll drop it for now if it turns out too much of a hassle.

Reply:
According to <https://github.com/vmactions/solaris-vm/issues/37>, there's only the CSW gcc available for this GH action. Therefore, I've trimmed the workflow down again to this one case for now.